tests/shared/nlu/training_data/test_training_data.py

```diff
@@ -691,6 +730,29 @@ def test_fingerprint_is_same_when_loading_data_again():
     assert td1.fingerprint() == td2.fingerprint()
 
 
+def test_fingerprint_is_different_when_lookup_table_has_changed(
+    monkeypatch: MonkeyPatch,
+):
+    from rasa.shared.importers.utils import training_data_from_paths
+
+    files = [
+        "data/test/lookup_tables/lookup_table.json",
+    ]
+
+    td1 = training_data_from_paths(files, language="en")
+    fingerprint1 = td1.fingerprint()
+
+    monkeypatch.setattr(
```

Comment:
why don't we write it to a `tmp_path` instead of mocking this? In my opinion we shouldn't mock unless we save time or it's not possible to test it otherwise